integration/cli/init.test.ts

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { beforeAll, describe, expect, it } from "@jest/globals";
+import * as path from "node:path";
+import * as fs from "node:fs/promises";
+import { Workdir } from "../helpers/workdir";
+import * as time from "../helpers/time";
+import * as pepr from "../helpers/pepr";
+
+const FILE = path.basename(__filename);
+const HERE = __dirname;
+
+describe("init", () => {
+  const workdir = new Workdir(`${FILE}`, `${HERE}/../testroot/cli`);
+
+  beforeAll(async () => {
+    await workdir.recreate();
+  });
+
+  it(
+    "gives command line help",
+    async () => {
+      const argz = "--help";
+      const res = await pepr.cli(workdir.path(), { cmd: `pepr init ${argz}` });
+      expect(res.exitcode).toBe(0);
+      expect(res.stderr.join("").trim()).toBe("");
+      expect(res.stdout.at(0)).toMatch("Usage: pepr init");
+    },
+    time.toMs("2m"),
+  );
+
+  it(
+    "creates new module using input args",
+    async () => {
+      const name = "flags-name";
+      const desc = "flags-desc";
+      const errb = "reject";
```

Comment:
```suggestion
      const errorBehavior = "reject";
```
`errb` means....? ErrorBehavior?